periodic-extent-spec.md

```diff
@@ -0,0 +1,136 @@
+
+## Schema
+
+- [JSON Schema](json-schema/schema.json)
+
+### Extent Object
+
+The object describes the spatio-temporal extents of the Collection. Both spatial and temporal extents are required to be specified.
+
+| Element  | Type                                              | Description                                                           |
+| -------- | ------------------------------------------------- | --------------------------------------------------------------------- |
+| spatial  | [Spatial Extent Object](#spatial-extent-object)   | **REQUIRED.** Potential *spatial extents* covered by the Collection.  |
+| temporal | [Temporal Extent Object](#temporal-extent-object) | **REQUIRED.** Potential *temporal extents* covered by the Collection. |
+
+#### Spatial Extent Object
+
+The object describes the spatial extents of the Collection.
+
+| Element | Type         | Description                                                          |
+| ------- | ------------ | -------------------------------------------------------------------- |
+| bbox    | \[\[number]] | **REQUIRED.** Potential *spatial extents* covered by the Collection. |
+
+**bbox**: Bounding Boxes of the assets represented by this Collection using either 2D or 3D geometries. Each outer array element can be a separate bounding box, but it is recommended to only use multiple bounding boxes if a union of them would then include a large uncovered area (e.g. the union of Germany and Chile).
+
+The length of the inner array must be 2*n where n is the number of dimensions. The array contains all axes of the southwesterly most extent followed by all axes of the northeasterly most extent specified in Longitude/Latitude or Longitude/Latitude/Elevation based on [WGS 84](http://www.opengis.net/def/crs/OGC/1.3/CRS84). When using 3D geometries, the elevation of the southwesterly most extent is the minimum depth/height in meters and the elevation of the northeasterly most extent is the maximum.
+
+The coordinate reference system of the values is WGS 84 longitude/latitude. Example that covers the whole Earth: `[[-180.0, -90.0, 180.0, 90.0]]`.  Example that covers the whole earth with a depth of 100 meters to a height of 150 meters: `[[-180.0, -90.0, -100.0, 180.0, 90.0, 150.0]]`.
+
+#### Temporal Extent Object
+
+The object describes the temporal extents of the Collection.
+
+| Element  | Type               | Description                                                           |
+| -------- | ------------------ | --------------------------------------------------------------------- |
+| interval | \[\[string\|[Periodic Interval](#periodic-interval)\|null]] | **REQUIRED.** Potential *temporal extents* covered by the Collection. |
+
+**interval**: Each outer array element can be a separate temporal extent, but it is recommended to only use multiple temporal extents if a union of them would then include a large uncovered time span (e.g. only having data for the years 2000, 2010 and 2020).
```

Comment:
I think linking to the OGC PR with this recommendation would be useful

Reply:
@jisantuc I just took it from here... https://github.com/radiantearth/stac-spec/blob/2c4ffa26691c1369598ccfdb8e1e6909bc782829/collection-spec/collection-spec.md#spatial-extent-object